metrics/instrumenting.go

```diff
@@ -158,7 +157,7 @@ func (m *influxMetrics) Write(bp influx.BatchPoints) error {
 
 // WriteLoop writes the data to the Influx DB.
 func (m *influxMetrics) WriteLoop(c <-chan time.Time) {
-	m.metrics.WriteLoop(c, m.influx)
+	m.metrics.WriteLoop(context.TODO(), c, m.influx)
```

Comment:
context.Background() is the same but it does not feel like if there is still something to do.